src/components/AutoSearch.tsx

```diff
@@ -68,34 +59,42 @@ export const AutoSearch: FC<AutoSearchProps> = ({
       open={open}
       onClose={() => {
         setOpen(false);
-        setOptions([]);
       }}
       onOpen={() => {
         setOpen(true);
       }}
       onChange={(_: object, value: string) => {
         if (value) {
           setValueFunction(value);
-          if (clearOnSelect) {
-            setDisplayValue(emptyValue);
-          }
+          setDisplayValue(emptyValue);
+          setOptions([]);
         }
       }}
-      onInputChange={(event: object, value: string) => {
-        value.length >= MINIMUM_SEARCH_CHARACTERS && options.length === 0 && open && setLoading(true);
-        open && value.length >= MINIMUM_SEARCH_CHARACTERS && onInputChange && onInputChange(event, value);
+      onInputChange={(event: any, value: string) => {
```

Comment:
sjekk ut reason som tredje parameter. da kan dere sjekke om reason === "input" istedenfor å sjekke eventet

Reply:
også kan dere kanskje bruke reason === "clear" for å clear'e options